client/src/pages/Home/Home.js

```diff
@@ -5,13 +5,14 @@ import Footer from "../component/footer/Footer";
 import "./Home.css";
 import Mission from "../component/Mission/Mission";
 import ContactUs from "../component/ContactUs/ContactUs";
+import HowIsWORK from "../component/HowItIsWork/HowIsWork";
```

Comment:
I'd go through everywhere that the name of this page "How it works" is being referenced and make sure that it is referred to in the same way: i.e. `HowItWorks` here, HowItWorks.js, howitworks.css etc etc. It really helps other developers if naming is consistent.

Reply:
yes, i should do what you said,  thank for advices

Reply:
i saw this different on the my layout page,my work is fine, it display well this different from Abed's work, we got the issue, i discuss this with Abed we decide to get solution for it